index.html

```diff
@@ -348,6 +389,51 @@ <h2>
           </li>
         </ol>
       </section>
+      <section>
+        <h2>
+          Preparing a new [=Web App Launch Client=]
+        </h2>
+        <p>
+          To <dfn>prepare a new web app launch client</dfn> given a
+          <a data-cite="appmanifest#dfn-processed-manifest">processed
+          manifest</a> |manifest| and |target URL:URL|, run the following steps:
+        </p>
+        <ol class="algorithm">
+          <li>Let |client| be the result of
+              <a href="https://html.spec.whatwg.org/multipage/browsers.html#creating-a-new-top-level-browsing-context">
+              creating a new top-level browsing context</a>.
+          </li>
+          <li><a data-cite="appmanifest#dfn-applied">Apply</a> |manifest| to
+              |client|.
+          </li>
+          <li>Run the steps to [=navigate=] |client| passing |target URL|.
+          </li>
+          <li>Return |client|.
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h2>
+         Processing [=unconsumed launch params=]
+        </h2>
+        <p>
+          The steps to <dfn>process unconsumed launch params</dfn> are as
```

Comment:
Take a LaunchQueue object _queue_. In the body, the "assigned launch consumer" should have "of _queue_".

(So we are explicit where we pull these objects from.)

Reply:
Done.